src/test/java/iudx/resource/server/database/async/AsyncServiceTest.java

```diff
@@ -213,7 +213,7 @@ public void failAsyncSearchForExistingRecordTest(VertxTestContext testContext) {
         .getRecord4RequestId(any());
     doAnswer(Answer -> Future.failedFuture("fail")).when(asyncServiceSpy).executePgQuery(any());
 
-    asyncServiceSpy.asyncSearch(requestId, sub, searchId, query);
+    asyncServiceSpy.asyncSearch(requestId, sub, searchId, query, "csv");
```

Comment:
Why hardcoded? format should be extracted from the consumed RMQ message.

Reply:
We might not be consuming the RMQ messages in the unit test class